src/pkg/services/m365/m365.go

```diff
@@ -97,7 +98,8 @@ func SiteIDs(ctx context.Context, m365Account account.Account) ([]string, error)
 // parseUser extracts information from `models.Userable` we care about
 func parseUser(item models.Userable) (*User, error) {
 	if item.GetUserPrincipalName() == nil {
-		return nil, errors.Errorf("no principal name for User: %s", *item.GetId())
+		return nil, clues.New("userable missing principal name").
```

Comment:
I personally prefer using "user" instead of "userable" as when this error lands in a cli output, "usersable" might be confusing to end users.